src/plugins/discover/public/context_awareness/profile_providers/example/example_root_profile/profile.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import {
+  EuiBadge,
+  EuiCodeBlock,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiTitle,
+} from '@elastic/eui';
+import { getFieldValue } from '@kbn/discover-utils';
+import React, { useState } from 'react';
+import { RootProfileProvider, SolutionType } from '../../../profiles';
+import { ExampleContextProvider } from '../example_context';
+
+export const createExampleRootProfileProvider = (): RootProfileProvider => ({
+  profileId: 'example-root-profile',
+  isExperimental: true,
+  profile: {
+    getRenderAppWrapper,
+    getCellRenderers: (prev) => (params) => ({
+      ...prev(params),
+      '@timestamp': (props) => {
+        const timestamp = getFieldValue(props.row, '@timestamp') as string;
+
+        return (
+          <EuiBadge color="hollow" title={timestamp} data-test-subj="exampleRootProfileTimestamp">
+            {timestamp}
+          </EuiBadge>
+        );
+      },
+    }),
+  },
+  resolve: (params) => {
+    if (params.solutionNavId != null) {
+      return { isMatch: false };
+    }
+
+    return { isMatch: true, context: { solutionType: SolutionType.Default } };
+  },
+});
+
+export const createExampleFallbackRootProfileProvider = (): RootProfileProvider => ({
```

Comment:
I introduced a separate `example-fallback-root-profile` which activates when in Solution contexts, in order for us to support functional tests in serverless without breaking all of the existing tests. Open to suggestions if someone has a better alternative.

Reply:
I like `example-solution-view-root-profile` 👍 Updated here: https://github.com/elastic/kibana/pull/197556/commits/2eede18fd5bb01c995340b36f9d59fe2ba0fc21b.